hivesim/hive.go

```diff
@@ -154,6 +155,40 @@ func (sim *Simulation) ClientEnodeURL(testSuite SuiteID, test TestID, node strin
 	return res, nil
 }
 
+type execInfo struct {
+	StdOut   string `json:"out"`
+	StdErr   string `json:"err"`
+	ExitCode int    `json:"code"`
+}
+
+// ClientRunProgram runs a command in a running client.
+func (sim *Simulation) ClientRunProgram(testSuite SuiteID, test TestID,
+	nodeid string, privileged bool, user string, cmd string) (stdOut string, stdErr string, exitCode int, err error) {
+
+	params := url.Values{}
+	params.Add("privileged", strconv.FormatBool(privileged))
```

Comment:
I don't want to add support for privileged exec because I just changed all container execution to unprivileged.
Also think it's not needed to specify the user. The API here should really just be

```
func (sim *Simulation) ClientExec(s SuiteID, t TestID, node string, cmd string, ...args string) (int, string, error)
```

Reply:
Still like having std-err and std-out separate. Combining them often becomes a mess, and I've spend more time debugging logs of just std-out or std-err, hiding the data I need, than I'd like. For a simple enode-like script it's not necessary, but when catching a panic after a complex keystore-export operation, it's great.

Removed the user/privilege options, and bundled the outputs.